ci/scripts/utils/publish_logs.py

```diff
@@ -46,7 +46,8 @@ def add_logs_to_gist(args, emcbot_gh):
 
     gist_files = {}
     for file in args.file:
-        file_content = file.read()
+        with open(file.name, 'r', encoding='latin-1') as file:
+            file_content = file.read()
```

Comment:
`file` is being redefined within the loop.

Reply:
I don't understand the issue with this?  Each file is opened to a new file object with this specific encoding an is read read.

Reply:
You're right, it doesn't matter since you aren't trying to work with the original file object.  Resolving.